podcasts/New Search/Views/Results/PodcastCarouselView.swift

```diff
@@ -0,0 +1,75 @@
+import SwiftUI
+import PocketCastsDataModel
+
+struct PodcastsCarouselView: View {
+    @EnvironmentObject var theme: Theme
+
+    let size: Double = 0.48
+    var body: some View {
+        ScrollView {
+            LazyHStack {
+                TabView {
+                    ForEach(0..<30) { i in
+                        GeometryReader { geometry in
+                            HStack(spacing: 10) {
+                                PodcastResultCell(podcast: Podcast.previewPodcast())
+
+                                PodcastResultCell(podcast: Podcast.previewPodcast())
+                            }
+                        }
+                    }
+                    .padding(.all, 10)
+                }
+                .frame(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height * 0.3)
+                .tabViewStyle(PageTabViewStyle(indexDisplayMode: .never))
+            }
+            ThemeableSeparatorView()
+                .padding(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 0))
```

Comment:
This is a bit of a nitpick, no need to specify all the edges here.

```suggestion
                .padding(.leading, 16)
```